functions.php

```diff
@@ -490,3 +490,32 @@ function ($canonical_url, $post) {
         return $options;
     });
 }
+
+// Enable Hide Page title by default when Pattern Layout is used
+add_action(
+    'transition_post_status',
+    function ($new_status, $old_status, $post): void {
+        // Check if the new status is 'publish' and the old status is not 'publish'
+        if ($new_status !== 'publish' || $old_status === 'publish') {
+            return;
+        }
+
+        // Check if the post type is 'page'
+        if ($post->post_type !== 'page') {
```

Comment:
This `if` statement could be merged with the previous one

Reply:
The ticket also specifies that `Besides pages, this should also be applied in Actions and Campaigns.` so I guess a more accurate check would be that it's not a Post?